doc/GettingStarted.md

```diff
@@ -263,6 +274,13 @@ In addition, Vitess requires the software and libraries listed below.
     . ./dev.env
     make build
     ```
+    
+    Note:
+        On OS X, if you can link the protobuf library back.
```

Comment:
Remove "if" here from sentence?

As I mentioned above, I think I would prefer if this is part of bootstrap.sh instead.